webpages/popup/style.css

```
@@ -20,8 +20,15 @@ body {
   display: flex;
   height: 60px;
 }
+.header-button {
+  right: 0;
+  position: fixed;
+  cursor: pointer;
+  padding: 17.5px;
```

```suggestion
  padding: 18px;
```
The settings icon in the popup is actually 1px smaller than the theme icon on the settings page.